media-sound/whipper/whipper-0.9.0.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python CD-DA ripper preferring accuracy over speed (forked from morituri)"
+HOMEPAGE="https://github.com/whipper-team/whipper"
+SRC_URI="https://github.com/whipper-team/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT+=" !test? ( test )"
+
+BDEPEND="test? ( dev-python/twisted[${PYTHON_USEDEP}] )"
+COMMON_DEPEND="media-libs/libsndfile"
+DEPEND="${COMMON_DEPEND}"
```

Comment:
so now that `DEPEND` is empty except for libsndfile, you can drop the `COMMON_DEPEND` and have `RDEPEND` inherit `DEPEND` instead. ;)